.circleci/config.yml

```diff
@@ -1,14 +1,195 @@
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: |
+          cat /tmp/workspace/env-vars >> $BASH_ENV
       - checkout
+      - run:
+          name: Loading all docker images
+          command: |
+            docker load -i /tmp/workspace/graphql-server.tar
+            docker load -i /tmp/workspace/subscriber.tar
+            docker load -i /tmp/workspace/self-deployer.tar
+            docker load -i /tmp/workspace/auth-server.tar
+            docker load -i /tmp/workspace/frontend.tar
+      - run:
+          name: Pushing graphql server
+          command: bash ./hack/push --TYPE=ci --REPONAME=${REPONAME} --IMGNAME=${GRAPHQL_SERVER_IMAGE} --IMGTAG=${IMGTAG}
+      - run:
+          name: Pushing frontend server
+          command: bash ./hack/push --TYPE=ci --REPONAME=${REPONAME} --IMGNAME=${FRONTEND_IMAGE} --IMGTAG=${IMGTAG}/hack/push --TYPE=ci --REPONAME=${REPONAME} --IMGNAME=${GRAPHQL_SERVER_IMAGE} --IMGTAG=${IMGTAG}
+      - run:
+          name: Pushing auth server
+          command: bash ./hack/push --TYPE=ci --REPONAME=${REPONAME} --IMGNAME=${AUTHENTICATION_SERVER_IMAGE} --IMGTAG=${IMGTAG}
+      - run:
+          name: Pushing subscriber server
+          command: bash ./hack/push --TYPE=ci --REPONAME=${REPONAME} --IMGNAME=${SUBSCRIBER_IMAGE} --IMGTAG=${IMGTAG}
+      - run:
+          name: Pushing self deployer
+          command: bash ./hack/push --TYPE=ci --REPONAME=${REPONAME} --IMGNAME=${SELF_DEPLOYER_IMAGE} --IMGTAG=${IMGTAG}
+
+  release:
+    machine:
+      image: circleci/classic:201808-01
+    environment:
+      IMGTAG: ci
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
       - run: |
-          source /opt/circleci/.nvm/nvm.sh && nvm alias default 10 && cd litmus-portal && make all
\ No newline at end of file
+          cat /tmp/workspace/env-vars >> $BASH_ENV
+      - checkout
+      - run:
+          name: Loading all docker images
+          command: |
+            docker load -i /tmp/workspace/graphql-server.tar
+            docker load -i /tmp/workspace/subscriber.tar
+            docker load -i /tmp/workspace/self-deployer.tar
+            docker load -i /tmp/workspace/auth-server.tar
+            docker load -i /tmp/workspace/frontend.tar
+      - run:
+          name: Docker images
+          command: docker images
+      - run:
+          name: Pushing graphql server
+          command: bash ./hack/push --TYPE=release --REPONAME=${REPONAME} --IMGNAME=${GRAPHQL_SERVER_IMAGE} --IMGTAG=${IMGTAG}
+      - run:
+          name: Pushing frontend server
+          command: bash ./hack/push --TYPE=release --REPONAME=${REPONAME} --IMGNAME=${FRONTEND_IMAGE} --IMGTAG=${IMGTAG}/hack/push --TYPE=ci --REPONAME=${REPONAME} --IMGNAME=${GRAPHQL_SERVER_IMAGE} --IMGTAG=${IMGTAG}
```

Comment:
validate the command once

Reply:
It a valid command. The pipelines are running in my local setup
https://app.circleci.com/pipelines/github/rajdas98/litmus-trial/79/workflows/76c9f4fe-b75b-4096-9a68-b8827b7866fc

Reply:
here two times push script is called one with type=ci and other with type=release

Reply:
It should be type=release only

Reply:
done